src/components/Calculator.js

```diff
@@ -1,39 +1,61 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+// import calculate from '../logic/calculate';
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Great job, to keep your code clean, please don't leave commented code (line 3).
- [ ] Rather than disabling linters, it's better to fix the error, kindly remove the code on line 4 and fix the error, you got this 💪.

Reply:
@Wes-Isaac  thanks, i have fixed that